README.md

```diff
@@ -6,9 +6,12 @@ This repository contains the Puppet modules for WSO2 Open Banking.
 
 1. Download the following zip files:<br>
```

Comment:
In doc, we ask to download the IS connector from here, https://apim.docs.wso2.com/en/4.0.0/assets/attachments/administer/wso2is-extensions-1.2.10.zip.
Is this applicable?

Reply:
There is a script to download IS connector in (obiam/files/merge-connector.sh). It will also extract and move the files to relevant location.